digitalfabrik/entitlementcard
# 1713: Fix MailNotification validation in CSV import

### Short description


Fix MailNotification validation in CSV import.

### Proposed changes



- Use default extension state if not provided
- Show `-` if string is empty

### Side effects



None.

### Testing


1. Go to http://localhost:3000/cards/import
2. Import bulk data
3. See the mail notification field error validation while cards can be created


### Resolved issues



Fixes: #1713
